index.html

```diff
@@ -8874,6 +8902,24 @@ <h2 id="WaveShaperNode">
               attribute therefore have no effect: the attribute must be set
               again in order to change the curve.
             </p>
+            <p>
+              To set the <code>curve</code> attribute, execute these steps:
```

Comment:
But the same could be said for the buffer in an `AudioBufferSourceNode`; the buffer is used "atomically" in some sense, with some odd behavior if the new buffer is much shorter than the previous.  The `ConvolverNode` also has this property of being used "atomically" in principle, but at least in Chrome, various parts of the response are used in multiple threads to speed up long convolutions.

So perhaps for consistency, make `WaveShaper` also have this constraint as well.